ioc_fanger/ioc_fanger.py

```diff
@@ -56,12 +57,13 @@ def defang(text):
 @click.argument("text", required=False)
 def cli_defang(text):
     """CLI interface for defanging indicators."""
-    stdin_text = click.get_text_stream("stdin")
-
+    
     if text:
         defanged_text = defang(text)
         print(defanged_text)
-    elif stdin_text:
+        return
```

Comment:
I recommend adding a newline here so it is easier for readers to see the two different paths we can take in this function (one path if there is text and another if there is stdin_text):

```suggestion
        return

```